### PR TITLE
Add ignoring SSL certificates

### PR DESCRIPTION
**Pull request by pawan-manishka**
Add ignoring SSL cerificates to use this JDBC driver with reporting tools like Eclipse BIRT...

### CODE REVIEW

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
  */
 package org.wso2.das.jdbcdriver.dasInterface;
 
+import javax.net.ssl.*;
```
Comment (anupama-pathirage) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: Avoid use imports with *. Always use the required imports only to have clean and unambiguous code.

Comment (pawan-manishka) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: removed it and added those required imports separately.

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java [hunk 2]
```diff
@@ -35,16 +36,56 @@
...
+        };
+
+        // Install the all-trusting trust manager
+        try {
+            SSLContext sc = SSLContext.getInstance("SSL");
+            sc.init(null, trustAllCerts, new java.security.SecureRandom());
+            HttpsURLConnection.setDefaultSSLSocketFactory(sc
+                    .getSocketFactory());
+            HttpsURLConnection.setDefaultHostnameVerifier(allHostsValid);
+
```
Comment (pawan-manishka) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: removed those lines.

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/jdbc/DASJDriver.java [hunk 3]
```diff
@@ -85,7 +86,9 @@ public boolean acceptsURL(String url) throws SQLException {
         DriverPropertyInfo[] propertyInfo = new DriverPropertyInfo[2];
         propertyInfo[0] = userProp;
         propertyInfo[1] = passwordProp;
+        System.out.println("property info");
```
Comment (pawan-manishka) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/jdbc/DASJDriver.java]: removed it.

File: das-jdbc/src/test/java/org/wso2/das/jdbc/DASJDriverTest.java [hunk 4]
```diff
@@ -65,7 +65,7 @@ public void isSampleSetup() {
     private boolean checkTableName() {
         boolean bRet = false;
         try {
-            String sTableName = "OVERUSED_DEVICES";
+            String sTableName = "OVERUSED_DEVICES";     //uni_sys
```
Comment (pawan-manishka) [inline das-jdbc/src/test/java/org/wso2/das/jdbc/DASJDriverTest.java]: removed it.

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java [hunk 5]
```diff
@@ -35,16 +36,56 @@
...
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (pawan-manishka) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: removed it.

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java [hunk 6]
```diff
@@ -35,16 +36,56 @@
...
         HttpURLConnection con = (HttpURLConnection) obj.openConnection();
+
+        /**
+         *ignoring SSL certificates
+         */
+        TrustManager[] trustAllCerts = new TrustManager[]{new X509TrustManager() {
+            public java.security.cert.X509Certificate[] getAcceptedIssuers() {
+                return null;
+            }
+
```
Comment (pawan-manishka) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: removed those lines.

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java [hunk 7]
```diff
@@ -53,4 +89,5 @@ public static String sendGet(String url, String user, String pass) throws Except
         }
         return (response.toString());
     }
+
```
Comment (anupama-pathirage) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java]: Remove unnecessary new lines

File: das-jdbc/src/main/java/org/wso2/das/jdbcdriver/jdbc/DASJConnection.java [hunk 8]
```diff
@@ -82,10 +82,10 @@ protected DASJConnection(String url, Properties info) throws SQLException {
         }
         this.connURL = url;
         this.connURLForDASTables = this.connURL + ServiceConstants.DAS_SERVICE_QUERIES.DAS_TABLE_NAMES_QUERY;
-
         if (info != null) {
             setProperties(info);
         }
+
```
Comment (anupama-pathirage) [inline das-jdbc/src/main/java/org/wso2/das/jdbcdriver/jdbc/DASJConnection.java]: Remove unnecessary new lines. Check all the places

**Pull request merged**